enumerables.rb

```diff
@@ -0,0 +1,166 @@
+# rubocop:disable Metrics/ModuleLength
+module Enumerable
+  def my_each
+    return to_enum(:my_each) unless block_given?
+
+    size.times do |x|
+      yield(self[x])
+    end
+    self
+  end
```

Comment:
# my_each
1. [ ] When a block is given when self is a range, returns the range itself after calling the given block once for each element in self
### Test 👉 `range.my_each(&block)`
Resulted in *NoMethodError* 
#
2. [ ] When a block is given when self is a range behaves like #each calls the given block once for each element in self
### Test 👉 `enum.my_each(&block)`

Resulted in *NoMethodError* 
#
3. [ ] When a block is given when self is a hash behaves like #each calls the given block once for each element in self
### Test 👉 `my_each_output`

|Expected|Got|
|:---:|:---:|
|`"[:a, 1][:b, 2][:c, 3][:d, 4][:e, 5]"`|`" "`|